clusterloader2/pkg/provider/provider.go

```diff
@@ -108,10 +108,10 @@ func NewProvider(initOptions *InitOptions) (Provider, error) {
 		configs[RootKubeConfigKey] = initOptions.KubemarkRootKubeConfigPath
 	}
 	switch initOptions.ProviderName {
-	case AKSName:
-		return NewAKSProvider(configs), nil
 	case AWSName:
 		return NewAWSProvider(configs), nil
+	case AzureName:
```

Comment:
```suggestion
	case AzureName, AKSName:
```

Should continue to handle `--provider=aks` for backward compatibility?

Reply:
I'm +1 on this (let's keep the rename of the provider type to `AzureProvider`)